src/praisonai-agents/praisonaiagents/tools/registry.py

```
@@ -409,54 +420,80 @@ def get_all(self) -> Dict[str, Union[BaseTool, Callable]]:
+                        warnings.warn(
+                            f"Tool '{ep.name}' is registered under the deprecated "
+                            f"entry-point group '{group}'. Publish under the "
+                            f"canonical '{CANONICAL_ENTRY_POINT_GROUP}' group instead.",
+                            DeprecationWarning,
+                            stacklevel=2,
+                        )
+
+                    seen.add(ep.name)
+                    count += 1
+                    logging.info(f"Discovered plugin tool: {ep.name}")
+                except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)